packages/jest-cli/src/cli/args.js

```diff
@@ -505,11 +505,11 @@ export const options = {
       'set up the testing environment before each test. ',
     type: 'array',
   },
-  setupTestFrameworkScriptFile: {
+  setupFilesAfterEnv: {
     description:
-      'The path to a module that runs some code to configure or ' +
-      'set up the testing framework before each test.',
-    type: 'string',
+      'A list of paths to modules that run some code to configure or ' +
```

Comment:
can we make the description consistent with `setupFiles`? I think this one sounds better and we could update `setupFiles`